mayastor-test/test_nexus.js

```diff
@@ -345,35 +345,9 @@ describe('nexus', function() {
     });
   });
 
-  it('should fail to publish with ShareProtocol None', done => {
-      let args = {
-          uuid: UUID,
-          share: mayastorProtoConstants.ShareProtocol.None
-      };
-
-      client.PublishNexus(args, (err, data) => {
-          if (!err) return done(new Error('Expected error'));
-          assert.equal(err.code, grpc.status.INVALID_ARGUMENT);
-          done();
-      });
-  });
-
-  it('should fail to publish with invalid ShareProtocol value of 100', done => {
-      let args = {
```

Comment:
Is this test no longer valid?

Reply:
I will partially reinstate, with the test commented out and a FIXME.
AFAICT without a None value in the enum, some javascript library changes the invalid value of 100 to 0 :-(. 
Don't know which library changes it and we probably do not want to change the lib either.
I checked this with Jan a few days ago, so not good but what we will have to settle for.
This test was introduced by me in a previous change in tjoshum/Mayastor so does not exist in openebs/Mayastor

Reply:
Fixed.
https://github.com/tjoshum/Mayastor/pull/8/commits/013a9ba2943286206917509264ae10d16ad2d038